server-common/src/main/java/org/apache/gravitino/server/web/JettyServer.java

```diff
@@ -299,7 +322,7 @@ private void initializeWebAppServletContextHandler() {
     if (warFile.isDirectory()) {
       // Development mode, read from FS
       servletContextHandler.setResourceBase(warFile.getPath());
-      servletContextHandler.setContextPath("/");
+      servletContextHandler.addServlet(DefaultServlet.class, "/");
```

Comment:
any reason changing this?

Reply:
We use a `web.xml` to initial the servletContextHandler. So need change this line.